align/pnr/checker.py

```diff
@@ -1,8 +1,9 @@
 from ..schema import constraint, types
 from ..cell_fabric import transformation
 
-def check_placement(placement_verilog_d):
+def check_placement(placement_verilog_d, scale_factor):
```

Comment:
@soneryaldiz This scaling issue has been fixed in another PR. A scaled version of placement_verilog_d is passed to this routine.

Reply:
Do you mean a version that is in units of layers.json?